tests/tools/groups/test_paths.py

```diff
@@ -117,16 +117,17 @@ def test_walk(setup_groups):
 
 
 def test_walk_with_invalid_path(clear_database_before_test):
+    """Test the ``GroupPath.walk`` method with invalid paths."""
     for label in ['a', 'a/b', 'a/c/d', 'a/c/e/g', 'a/f', 'bad//group', 'bad/other']:
-        orm.Group.objects.get_or_create(label, type_string=orm.GroupTypeString.USER.value)
+        orm.Group.objects.get_or_create(label)
     group_path = GroupPath()
-    assert [c.path for c in sorted(group_path.walk())
-           ] == ['a', 'a/b', 'a/c', 'a/c/d', 'a/c/e', 'a/c/e/g', 'a/f', 'bad', 'bad/other']
+    expected = ['a', 'a/b', 'a/c', 'a/c/d', 'a/c/e', 'a/c/e/g', 'a/f', 'bad', 'bad/other']
+    assert [c.path for c in sorted(group_path.walk())] == expected
 
 
 def test_walk_nodes(clear_database):
```

Comment:
```suggestion
def test_walk_nodes(clear_database_before_test):
```

Unrelated to this PR, but should the default for `clear_database` be `clear_database_before_test`? I can't really think of a use case where it is ever good to be doing `clear_database_after_test`?